tests/tt_eager/python_api_testing/unit_testing/misc/test_binary_ops_ttnn.py

```diff
@@ -0,0 +1,127 @@
+# SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+
+# SPDX-License-Identifier: Apache-2.0
+
+import torch
+import pytest
+import ttnn
+from tests.tt_eager.python_api_testing.unit_testing.backward_ops.utility_funcs import data_gen_with_range, compare_pcc
```

Comment:
this file will go under` tests/ttnn/unit_tests/operations/composite`
and use `tests/ttnn/unit_tests/operations/backward/utility_funcs.py` for import